wheel_encoders:
  debug_mode:           False
  serial_port:          "/dev/pts/8"
  send_command_retries: 3
  encoder_timeout_ms:   12
  loop_frequency:       30
  baud_rate:            115200
  right_wheel:          ['wheel_Right_Front_joint', 'wheel_Right_Middle_joint', 'wheel_Right_Back_joint']
  left_wheel:           ['wheel_Left_Front_joint', 'wheel_Left_Middle_joint', 'wheel_Left_Back_joint']
  right_addr:           [ 128, 129, 130 ] # note: implement
  left_addr:            [ 128, 129, 130 ] # note: implement